jme3-core/src/com/jme3/gde/core/util/TaggedSpatialFinder.java

```diff
@@ -0,0 +1,68 @@
+package com.jme3.gde.core.util;
+
+import com.jme3.scene.Spatial;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Finds a previously marked spatial in the supplied root Spatial, creates
+ * the name and path to be looked for from the given needle Spatial.
+ */
+public class TaggedSpatialFinder {
+    
+    private static final Logger LOGGER =
+            Logger.getLogger(TaggedSpatialFinder.class.getName());
+    
+    public TaggedSpatialFinder() {
```

Comment:
We could honor this Codacy opinion, an empty default cosntructor is not gaining us anything

Reply:
I think I added it because Codacy complained there was no default constructor..